### ISSUE TITLE
Width of table of contents

### ISSUE THREAD

**Issue by BonaBeavis** [RDFLib/pyLODE#170]
With longer class names the line breaks reduce readability, because there is no separating element between classes. A solution would either be to increase the width of the table of contents or add a visual separation between the class names.
![image](https://user-images.githubusercontent.com/3330855/176229398-05a6d148-6cc0-4cfa-9e44-b24c15e775cb.png)

**Comment by BonaBeavis** [comment]
Also, some kind of sorting would be nice...

**Comment by nicholascar** [comment]
Yep, I see this issue myself in some of my ontologies. I'll try and actually put some effort in here shortly!
